R/XcmsExperiment-functions.R

```diff
@@ -859,8 +863,9 @@
 }
 
 #' Function to help extracting the *old* MChromatograms and XChromatograms
-#' from an XcmsExperiment. This is based on the code for XCMSnExp objects,
-#' but there could be room for improvement.
+#' from an XcmsExperiment.
+#'
+#' @param object `XcmsExperiment` or `XCMSnExp` object.
 #'
 #' @noRd
 .xmse_extract_chromatograms_old <- function(object, rt, mz, aggregationFun,
```

Comment:
this function was modified to improve performance of the `chromatogram,XcmsExperiment` call.